docs/Configuration.md

```diff
@@ -87,55 +90,40 @@ data:
   NAMESPACES: "default"    # ""
 ```
 
-### Authentication to Remote Services
-
-Pelorus exporters make use of `personal access tokens` when authentication is 
-required.  It is recommended to configure the Pelorus exporters with authenticaion
-via the `TOKEN` key to avoid connection rate limiting and access restrictions.
+### Authentication to remote services
+Pelorus exporters use `personal access tokens` when authentication is required. It is recommended to configure the Pelorus exporters with authentication using the `TOKEN` key to avoid connection rate limiting and access restrictions.
 
-More information about personal access tokens:
-* [Github Personal Access Tokens](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token)
+See [GitHub Personal Access Tokens](https://docs.GitHub.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) for more information about personal access tokens.
 
 * [Jira / Bitbucket Personal Access Tokens](https://confluence.atlassian.com/bitbucketserver/personal-access-tokens-939515499.html)
-
 * [Gitlab Personal Access Tokens](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html)
-
 * [Microsoft Azure DevOps Tokens](https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate?view=azure-devops&tabs=Windows)
-
 * [Gitea Tokens](https://docs.gitea.io/en-us/api-usage/#generating-and-listing-api-tokens)
 
-To store a personal access token securely and to make the token available to Pelorus use
-Openshift secrets. Pelorus can utilize secrets for all of the exporters. The following
-is an example for the committime exporter.
+Use Openshift secrets to store a personal access token securely and make it available to Pelorus to use for all of the exporters. The following is an example for the committime exporter in GitHub.
 
-A simple Github example:
 ```shell
-oc create secret generic github-secret --from-literal=TOKEN=<personal access token> -n pelorus
+oc create secret generic GitHub-secret --from-literal=TOKEN=<personal access token> -n pelorus
```

Comment:
this should be github-secret, because it's name of the secret within OpenShift cluster that is used in other places, so the capital letters may cause issues.